Protocols/CardsDataSource.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CardsDataSource.swift
+//  Protocols
+//
+//  Created by Felipe Gil on 2021-09-24.
+//
+
+import Foundation
+
+protocol CardsDataSource {
+    var playerName: String? { get set }
+    var cards: [String] { get set }
```

Comment:
Cards also seem to be set internally only as well.

Reply:
Fixed